corehq/apps/change_feed/pillow.py

```diff
@@ -9,16 +9,18 @@
 from corehq.apps.change_feed.topics import get_topic
 from couchforms.models import all_known_formlike_doc_types
 import logging
+from pillowtop.checkpoints.manager import PillowCheckpoint, get_django_checkpoint_store
 from pillowtop.couchdb import CachedCouchDB
 from pillowtop.listener import PythonPillow
 
 
 class ChangeFeedPillow(PythonPillow):
 
-    def __init__(self, couch_db, kafka):
+    def __init__(self, couch_db, kafka, checkpoint):
         super(ChangeFeedPillow, self).__init__(couch_db=couch_db)
         self._kafka = kafka
         self._producer = KeyedProducer(self._kafka)
+        self._checkpoint = checkpoint
```

Comment:
That is sneaky. I'm assuming this isn't the final design right. Probably
fine to leave it for now.

I was also thinking that a design doc or readme would be helpful for the
new structure. Even just a diagram.